public/locales/pt-BR/translations.json

```diff
@@ -52,10 +53,22 @@
       "invalidToken": "Token inválido. Tente novamente",
       "loginSessionExpired": "Sua sessão de login expirou, por favor tente novamente"
     },
+    "ttlSelect": {
+      "title": "TTL do Token de Autenticação",
+      "1day": "1 Dia",
```

Comment:
could make sense to have only `days` and `day` strings translated leaving out the numbers

Reply:
Good idea! This may fit well for almost all available languages except russian.
<img width="336" alt="ru_days" src="https://github.com/user-attachments/assets/861f8ed5-082c-4c8c-a134-05c8aa92dfa8" />

It has 2 values for plurals, so there will be 3 translated values (day, days_1, days_2) to unify and cover this case. Updated this way.